src/routes/api/sections/querySection.ts

```diff
@@ -0,0 +1,37 @@
+import { Request, Response } from 'express';
+import sectionModel from '../../../models/section.model';
+import timeblockModel from '../../../models/timeblock.model';
+
+const querySection = async (req: Request, res: Response): Promise<void> => {
+    // Retrieves the room section with the id given in the link
+    const sectionInformation = await sectionModel.findOne({ _id: req.params.id }).catch((error) => {
+        res.status(404).json({ error });
```

Comment:
```suggestion
        res.status(500).json({ error });
```

I think this would make more sense as an internal server error.